src/main/java/org/openrewrite/java/testing/assertj/JUnitAssertTrueToAssertThat.java

```diff
@@ -23,14 +23,17 @@
 import org.openrewrite.java.JavaParser;
 import org.openrewrite.java.JavaTemplate;
 import org.openrewrite.java.MethodMatcher;
-import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.search.UsesMethod;
 import org.openrewrite.java.tree.Expression;
 import org.openrewrite.java.tree.J;
 import org.openrewrite.java.tree.TypeUtils;
 
 import java.util.List;
 
 public class JUnitAssertTrueToAssertThat extends Recipe {
+
+    private static final MethodMatcher ASSERT_TRUE_MATCHER = new MethodMatcher("org.junit.jupiter.api.Assertions" + " assertTrue(boolean, ..)");
```

Comment:
No need to concatenate Strings here; please apply throughout; ideally we enforce this with a recipe as well.
```suggestion
    private static final MethodMatcher ASSERT_TRUE_MATCHER = new MethodMatcher("org.junit.jupiter.api.Assertions assertTrue(boolean, ..)");
```